### PR TITLE
Community and Support menus and content: update

### PR DESCRIPTION
**Pull request by grahamperrin**
Bugs <https://bugs.freebsd.org/261258> and <https://bugs.freebsd.org/260968>, respectively: 

> Q&A Link on home page points to wrong web site

> Support: Web resources (page update)

### CODE REVIEW

**Comment by grahamperrin** [comment]
From [`cc37892` (#187)](https://github.com/freebsd/freebsd-doc/pull/187/commits/cc37892e2b0aef5f049195aa2b158f72b2711aa1): 

> 𡀦… heading level distinctions … are not visible at the FreeBSD site.

New bug report (I thought I had reported this months ago): 

* [271477 – Distinctions between heading levels are lost in areas of www.freebsd.org](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=271477)

**Comment by grahamperrin** [comment]
When I click the **Support** menu, I expect support that's FreeBSD-specific, or FreeBSD-oriented.

Things such as [Related Operating System Projects](https://www.freebsd.org/support/webresources/#related) really don't belong. (How can a 1996 release of an operating system that is **not** FreeBSD-based provide support for use of FreeBSD 12, 13, and 14? And so on.)

Let's see what's at The FreeBSD Foundation site, and deduplicate.

**Comment by igalic** [comment]
https://bugs.freebsd.org/bugzilla/261258 -> 404

File: website/content/en/community/_index.adoc [hunk 1]
```diff
@@ -7,31 +7,31 @@ include::shared/en/urls.adoc[]
...
-Beyond officially run community spaces, there are https://www.freebsd.org/usergroups/[User Groups] in {{< get-usergroups-info "countries" >}} countries, along with highly active FreeBSD communities on https://twitter.com/search?q=freebsd[Twitter], https://stackoverflow.com/questions/tagged/freebsd[StackOverflow], https://serverfault.com/questions/tagged/freebsd[ServerFault], and https://www.meetup.com/topics/freebsd/[MeetUp.com], among many other technology-related spaces.
+The https://wiki.freebsd.org/[FreeBSD wiki] includes information about development and related projects.
 
-The https://wiki.freebsd.org/[FreeBSD wiki] contains information about development and related projects.
+Non-official spaces include: 
 
-== Video Content
-
-On YouTube, we host an official link:https://www.youtube.com/FreeBSDProject[FreeBSD] channel with developer summits and "office hours" events, and a link:https://www.youtube.com/bsdconferences[BSD Conferences] channel with full taped presentations from FreeBSD technical conferences.
+* https://www.freebsd.org/usergroups/[User Groups] in {{< get-usergroups-info "countries" >}} countries
```
Comment (lwhsu) [inline website/content/en/community/_index.adoc:35]: again a relative link can be used.

**Comment by grahamperrin** [comment]
> … 404

Fixed,thanks.

**Pull request closed without merging**